checker/build.gradle

```diff
@@ -18,13 +27,29 @@ dependencies {
     // file. :-( So update it and re-build it locally when updating this.
     implementation 'org.plumelib:reflection-util:0.2.2'
 
+    // Object Construction Checker AutoValue + Lombok support
+    implementation "com.google.auto.value:auto-value-annotations:1.7.4"
```

Comment:
Like we did with the Returns Receiver Checker we likely want to remove this `implementation` dependence and use reflection to identify AutoValue annotations instead

Reply:
Thanks for pointing that out. I removed the dependency.